python/mlc_llm/quantization/group_quantization.py

```diff
@@ -983,3 +980,14 @@ def forward(self, x: nn.Tensor, indptr: nn.Tensor) -> nn.Tensor:  # pylint: disa
                 indptr_dtype=indptr.dtype,
                 group_size=self.group_size,
             )
+        # TODO(@team): this is from upstream. commented out for now.
```

Comment:
cc. @csullivan @JosephTheOctonaut @vinx13

Reply:
we can clean up fp8 here as well